src/components/_DeploymentAnnotation/DeploymentAnnotation.js

```diff
@@ -64,14 +64,17 @@ const InfoLine = styled.span`
   overflow: hidden;
 `;
 
-class DeploymentAnnotations extends React.PureComponent {
-  constructor(props) {
-    super(props);
+const NoLink = styled.span`
+  cursor: default;
```

Comment:
Is this needed, `span` should have default anyway?

Reply:
The default value is normally `inherit` I think and in case of `TimeTravel`, the parent element has cursor `move` - without this line, we can never be sure it's properly enforced.